00-HTML-CSS-basics/ex1.VI.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
This looks like inconsistent for me. Should this be `doctype` or `DOCTYPE`. Most of things in development are case sensitive.